examples/05-produce-cloud/src/main.rs

```diff
@@ -0,0 +1,47 @@
+//! A minimal example showing how to produce messages on Fluvio
+//!
+//! Before running this example, make sure you have created a topic
+//! named `simple` with the following command:
+//!
+//! ```text
+//! $ fluvio topic create simple
+//! ```
+//!
+//! Run this example using the following:
+//!
+//! ```text
+//! $ cargo run --bin produce-cloud
+//! Sent simple record: Hello, Fluvio!
+//! ```
+//!
+//! After running this example, you can see the messages that have
+//! been sent to the topic using the following command:
+//!
+//! ```text
+//! $ fluvio consume simple -B -d
+//! Hello, Fluvio!
+//! ```
+
+use fluvio::config::ConfigFile;
+use fluvio::RecordKey;
+
+#[async_std::main]
+async fn main() {
+    if let Err(e) = produce().await {
+        println!("Produce error: {:?}", e);
+    }
+}
+
+async fn produce() -> Result<(), fluvio::FluvioError> {
+    let config = ConfigFile::load(None).unwrap();
+    let fluvio_config = config.config().cluster.get("cloud").unwrap();
```

Comment:
I think that we should mention in some place that we need to make sure that this is created, by calling first `fluvio cloud login` with the infinyon.cloud credentials